index.html

```diff
@@ -4,484 +4,528 @@
               </a>
 
-              <a href="#" class="content-container__job-blocks job-block">
+              <a href="#" class="jobs-group__job-blocks job-block">
                 <div class="job-block__circle">
                   <img
+                    class="job-block__img"
                     src="./src/icons/human-resourse.svg"
                     alt="human resourse"
                   />
+
+                  <img
+                    class="job-block__img_active"
+                    src="./src/icons/human-resourse-active.svg"
+                    alt="human resourse"
+                  />
                 </div>
-                <h2 class="job-block__title">Human Resource</h2>
+                <p class="job-block__title">Human Resource</p>
                 <p class="job-block__subtitle">35 Job Vacancy</p>
               </a>
 
-              <a href="#" class="content-container__job-blocks job-block">
+              <a href="#" class="jobs-group__job-blocks job-block">
                 <div class="job-block__circle">
                   <img
+                    class="job-block__img"
                     src="./src/icons/art&illustration.svg"
                     alt="art&illustration"
                   />
+
+                  <img
+                    class="job-block__img_active"
+                    src="./src/icons/art&illustration-active.svg"
+                    alt="art&illustration"
+                  />
                 </div>
-                <h2 class="job-block__title">Art & Illustration</h2>
+                <p class="job-block__title">Art & Illustration</p>
                 <p class="job-block__subtitle">52 Job Vacancy</p>
               </a>
             </div>
           </div>
 
           <!-- Sign up block -->
-          <div class="second-section__sign-up-block">
+          <div class="categories__sign-up">
             <!-- Sign up block left wrapper -->
-            <div class="sign-up-block__left-wrapper">
+            <div class="sign-up__info">
               <img
-                class="left-wrapper_dots"
+                class="sign-up__dots"
                 src="./src/backgound-svg/dots-design.svg"
                 alt="dots"
               />
-              <h2 class="sign-up-block__title title">Get Start New Job</h2>
-              <p class="sign-up-block__subtitle">
+              <h2 class="sign-up__title title">Get Start New Job</h2>
+              <p class="sign-up__subtitle">
                 Create an account so you can get job information that suits you,
                 and you can apply the salary your demand.
               </p>
 
               <!-- Sign up button -->
-              <div class="sign-up-block__button">
-                <a class="buttons" href="#"> Create an Account </a>
+              <div class="sign-up__button">
+                <button class="button" href="#">Create an Account</button>
               </div>
```

Comment:
```suggestion
                <button class="button" href="#">Create an Account</button>
```
You don't need an extra wrapper. Reach this class in CSS by `.sign-up .button {}`